src/f5_tts/infer/utils_infer.py

```diff
@@ -359,7 +466,7 @@ def infer_batch_process(
     nfe_step=32,
     cfg_strength=2.0,
     sway_sampling_coef=-1,
-    speed=1,
+    speed=0.8,
```

Comment:
better go for the linear estimated duration operation part (a factor there) if serves as default